mercadopago/resources/order.py

```diff
@@ -33,3 +32,131 @@ def create(self, order_object, request_options=None):
             raise ValueError("Param order_object must be a Dictionary")
 
         return self._post(uri="/v1/orders", data=order_object, request_options=request_options)
+
+    def get(self, order_id, request_options=None):
+        """[Click here for more info](https://www.mercadopago.com.br/developers/en/reference/order/online-payments/get-order/get ) # pylint: disable=line-too-long
```

Comment:
[req] Remover `.br` para a URL ficar global
```suggestion
        """[Click here for more info](https://www.mercadopago.com/developers/en/reference/order/online-payments/get-order/get ) # pylint: disable=line-too-long
```